modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -6,21 +6,19 @@
 [id="about-lvmcluster_{context}"]
 = About the LVMCluster custom resource
 
-After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worker node.
-
 You can configure the `LVMCluster` CR to perform the following actions:
 
 * Create LVM volume groups that you can use to provision persistent volume claims (PVCs).
 * Configure a list of devices that you want to add to the LVM volume groups. 
 * Configure the requirements to select the nodes on which you want to create an LVM volume group, and the thin pool configuration for the volume group.
 * Force wipe the selected devices.
 
-You can create an `LVMCluster` CR using the OpenShift CLI (`oc`) or the {product-title} web console.
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worker node.
```

Comment:
```suggestion
After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) using CLI or web-console.
```

Reply:
We can skip the word 'worker node', as user don't have to specifically perform any action on 'worker node', it's handled by LVMS.

Reply:
Updated as per the suggestion.